src/components/menu/menu-item/menu-item.test.tsx

```diff
@@ -0,0 +1,843 @@
+import React from "react";
+import { fireEvent, render, screen, within } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+import { MenuItem } from "..";
+import {
+  testStyledSystemLayout,
+  testStyledSystemFlexBox,
+  testStyledSystemPadding,
+} from "../../../__spec_helper__/test-utils";
+import StyledMenuItemWrapper from "./menu-item.style";
+import MenuContext, { MenuContextProps } from "../menu.context";
+import Icon from "../../icon/icon.component";
+import menuConfigVariants from "../menu.config";
+import IconButton from "../../icon-button";
+import Search from "../../search";
+import SubmenuContext from "../__internal__/submenu/submenu.context";
+import Logger from "../../../__internal__/utils/logger";
+
+// mock Logger.deprecate it isn't triggered while running the tests that use Search,
+const loggerSpy = jest.spyOn(Logger, "deprecate");
```

Comment:
I'd prefer to tweak how it's utilised here rather than remove it altogether as we run the risk of bloating our test outputs as we have several long standing deprecation warnings. I do think it's worth us revisiting this though as we could potentially implement an internal test helper here to essentially white list the logs we want to ignore vs legitimate ones we should be aware of etc

Reply:
I've decided to just remove as it's just the one log here